### PR TITLE
Move 10234 to Appium

### PR DESCRIPTION
**Pull request by PureWeen**

### CODE REVIEW

File: src/Controls/samples/Controls.Sample.UITests/Issues/Issue20439.xaml.cs [hunk 1]
```diff
@@ -3,8 +3,26 @@
 
 namespace Maui.Controls.Sample.Issues
 {
-	[XamlCompilation(XamlCompilationOptions.Compile)]
 	[Issue(IssueTracker.Github, 20439, "[iOS] Double dash in Entry or Editor crashes the app", PlatformAffected.iOS)]
+	public class Issue20439Test : ContentPage
```
Comment (PureWeen) [inline src/Controls/samples/Controls.Sample.UITests/Issues/Issue20439.xaml.cs:7]: The runners don't really work with Shell right now so I changed this one to swap in the main page as part of the test

**Review by jsuarezruiz — DISMISSED** [review]
The current failing tests on CI seems unrelated.

**Comment by PureWeen** [comment]
UI Tests are all passing

**Pull request merged**